cliquet/resource/__init__.py

```diff
@@ -580,8 +580,17 @@ def postprocess(self, result, action=ACTIONS.READ):
         # when it's added
         if action != ACTIONS.READ:
             # the collection has been changed
+            if not isinstance(result, list):
+                result = [result]
+            impacted = {}
+            if action == ACTIONS.CREATE:
+                impacted = [{'new': r} for r in result]
+            elif action == ACTIONS.DELETE:
+                impacted = [{'old': r} for r in result]
+            elif action == ACTIONS.UPDATE:
+                impacted = [{'new': r, 'old': old} for r in result]
```

Comment:
I might be wrong but I think in that case result is a list with only one entry since we cannot update multiple records at once yet.